Filters Screen

mr-wes7694 opened the pull request:
Duplicated browse screen components to make filters screen. Buttons are selectable instead of navigable.

Commit:
Working on Filters Screen

Duplicated browse screen components to make filters screen. Buttons are selectable instead of navigable.

Commit:
Filters Screen

Created duplicates of browse screen components to make selectable buttons.

BobDresn committed:
Merge branch 'dev' into wes/filters-screen

BobDresn reviewed (commented):
Can you just change the name of the button instead of moving the entire button across like we talked about?

Commit:
Filters Screen

Duplicated browse screen components to make filters screen. Buttons are selectable instead of navigable.

Commit:
Filters Screen

Duplicated browse screen components to make filters screen. Buttons are selectable instead of navigable.

mr-wes7694 commented:
> Can you just change the name of the button instead of moving the entire button across like we talked about?

That was what I tried to do. However it seems its still including a prior change I made.

Commit:
Filters Screen

Duplicated browse screen components to make filters screen. Buttons are selectable instead of navigable.

Commit:
Filters Screen

Duplicated browse screen components to make filters screen. Buttons are selectable instead of navigable.

Commit:
Filters Screen

Duplicated browse screen components to make filters screen. Buttons are selectable instead of navigable.

Commit:
Filters Screen

Duplicated browse screen components to make filters screen. Buttons are selectable instead of navigable.

Commit:
Filters Screen

Duplicated browse screen components to make filters screen. Buttons are selectable instead of navigable.

byte2pixel reviewed (approved):
Looks good, I think the category stuff will need reworking once we have filters saving to the user when they change.

Is the idea going to be every click will save the update to the user metadata? If there is no save button?
